graphql-dgs/src/main/kotlin/com/netflix/graphql/dgs/internal/DataFetcherInvoker.kt

```diff
@@ -62,10 +72,40 @@ class DataFetcherInvoker internal constructor(
             args[idx] = resolvers.resolveArgument(parameter, environment)
         }
 
-        return if (isSuspending) {
-            CoroutinesUtils.invokeSuspendingFunction(bridgedMethod, dgsComponent, *args)
+        return ReflectionUtils.invokeMethod(bridgedMethod, dgsComponent, *args)
+    }
+
+    private fun invokeKotlinMethod(kFunc: KFunction<*>, dfe: DataFetchingEnvironment): Any? {
+        val parameters = kFunc.parameters
+        val argsByName = CollectionUtils.newLinkedHashMap<KParameter, Any?>(parameters.size)
+
+        val paramSeq = if (parameters[0].kind == KParameter.Kind.INSTANCE) {
```

Comment:
Could you explain what lines #82 and #83 are doing?

Reply:
We are checking if the `KFunction` takes an instance parameter, and supplying it if so; that is, the target object we want to call the method on, in this case the DGS component. That said, I can't really think if a scenario where we won't have an instance parameter, but we are kind of forced to deal with the possibility.